symfony/webpack-encore-bundle/1.0/webpack.config.js

```diff
@@ -1,5 +1,9 @@
 var Encore = require('@symfony/webpack-encore');
 
+// Optional: sync the NODE_ENV value with the current Webpack Encore environment
+// It’s useful if you need to read the Webpack Encore environment from some other config file
+// process.env.NODE_ENV = Encore.isProduction() ? 'production' : 'development';
```

Comment:
this should be moved after the `Manually configure the runtime environment` case